tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventParseMethodContract.scala

```diff
@@ -1195,9 +1195,9 @@ trait LinagoraCalendarEventParseMethodContract {
            |                "uid": "037aaad3-17c9-47c8-bd6b-f2cbfe925ef7",
            |                "title": "MOB: integration tests",
            |                "start": "2023-03-09T14:00:00",
-           |                "end": "2023-03-09T14:00:00",
+           |                "end": "2023-03-10T14:00:00",
```

Comment:
Note:
In the new ical4j version, if both DTEND and DURATION are missing in the ICS file, the default "endDate" is set to startDate + 1 day.
```
// If "DTSTART" is a DATE, then the event's duration is one day (see: RFC 5545, 3.6.1 Event Component)
```